olive/passes/pass_config.py

```diff
@@ -93,6 +94,19 @@ def get_data_config(required: Optional[bool] = False):
     return data_config
 
 
+def get_eval_config(required: Optional[bool] = False):
+    eval_config = {
+        "evaluator": PassConfigParam(
```

Comment:
@jambayk , @trajepl , how should we differentiate the evaluator with https://microsoft.github.io/Olive/overview/options.html#passes-information if we make it as part of default config?

Reply:
Yes, otherwise, the config for one passes will be like:
```json
"passes": {
    "quantization": {
        "type": "IncQuantization",
        "config": {
                "evaluator": "common_evaluator" <---------------
            }

         "evaluator": "common_evaluator" <---------------
    }
}
```

The two same evaluators look a bit confusing.

Reply:
After offline syncing, @ftian1 , we suggest to implement it as Mike's suggestion to directly **pass the metric you needed with config way**. It will be like:

```json
"passes": {
    "quantization": {
        "type": "IncQuantization",
        "config": {
            "metric": {
                    "name": "accuracy",
                    "type": "accuracy",
                    "sub_types": [
                        {"name": "accuracy_score", "priority": 1}
                    ],
                    "user_config":{
                        "post_processing_func": "post_process",
                        "user_script": "user_script.py",
                        "dataloader_func": "create_dataloader",
                        "batch_size": 1
                    }
            }
        }
         "evaluator": "common_evaluator" 
    }
}
```

Reply:
Then we can remove the flag of `_requires_eval_config` as it is not common flag for all passes like Mike's comment.